process-manager.js

```diff
@@ -88,38 +93,46 @@ class ProcessManager {
 	release(process) {
 		process.release();
 	}
+
 	send() {
 		if (!this.processes.length) {
 			return Promise.resolve(this.receive.apply(this, arguments));
 		}
 
+		// Prevents the arguments object from leaking.
+		let args = [];
+		for (let i = 0; i < arguments.length; i++) {
+			args.push(arguments[i]);
+		}
```

Comment:
It'd be nice if we could make up our mind on this so I'm not sedding back and forth